authorization/auth_cloudSolutionProvider.md

```diff
@@ -0,0 +1,62 @@
+
+The initial steps required here follow most of the same steps used to register and configure a multi-tenant application:
+
+1. Register your application in your Partner tenant. You can use the <a href="https://portal.azure.com" target="_newtab">Azure Portal</a> for this. To function as a PMA, an application must be configured as a multi-tenant app.
+
+2. Configure your multi-tenant app with the *required permissions* it needs using a least privileged approach.
+
+3. Finally grant your Partner Managed App those configured permissions for all your customers. You can so this by adding the **servicePrincipal** that represents the app to the AdminAgents group in your Partner tenant, using Azure AD powershell V2. You can download and install Azure AD PowerShell V2 from <a href="https://www.powershellgallery.com/packages/AzureAD" target="_newtab">here</a>.  Follow these steps to find the AdminAgents group, the **servicePrincipal** and add it to the group.
+
+    1. Open a PowerShell session and connect to your partner tenant by entering your admin credentials into the sign-in window.
+    ```code
+    > Connect-AzureAd
+    ```
+    2. Next find the group that represents the AdminAgents. 
+    ```code
+    > $group = Get-AzureADGroup -Filter "displayName eq 'AdminAgents'"  
+    ```
+    3. Find the service principal that has the same *appId* as your app.
+    ```code
+    > $sp = Get-AzureADServicePrincipal -Filter "appId eq '{yourAppsAppId}'"
+    ```
+    4. Finally add the service principal to the AdminAgents group.
+    ```code
+    > Add-AzureADGroupMember -ObjectId $group.ObjectId -RefObjectId $sp.ObjectId
+    ```
+
+## Token acquisition flows
+Partner managed apps are identical to regular multi-tenant apps in virtually all respects. As such, the token acquisition flows <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a> or the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service" target="_newtab">service to service client credentials flow</a> are exactly the same.
+
+Apart from pre-consented access to all your customer tenants, there is one other capability afforded to partner managed apps. It allows for your agents to use your app to access your customers' tenant data (delegated admin privileges). Conceptually your agent signs in to your app with their user credentials from your partner tenant, but your app then acquires an access token to one of your partner managed customers' tenants, and uses that to call Microsoft Graph. This is a slight variation on the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a>. 
```

Comment:
The conceptual section is a lot of information to grasp in one sentence.  It might work better as a list.

Conceptually it works like this:
1. Your agent signs in to your app using their user credentials issued from your managed partner tenant. (I tried to make this clearer. Is this right? It's confusing.)
2. Your app requests an access token for the intended managed partner tenant.
3. Your app uses the access token to call Microsoft Graph APIs for the intended managed partner tenant.